babel.config.js

```diff
@@ -3,9 +3,7 @@ module.exports = {
     [
       '@babel/preset-env',
       {
-        targets: {
-          node: 'current',
-        },
+        "targets": "> 0.25%, not dead",
```

Comment:
This probably makes more sense than targeting node, right? Technically our Storyshots run in node but... I'm guessing browsers are mostly going to be more conservative?

Reply:
Yeah, I think that seems reasonable - the target is certainly the browser, not node.